pom.xml

```diff
@@ -99,4 +99,27 @@
             </plugin>
         </plugins>
     </build>
+    <!--
+        @todo #16 Remove this declaration when parent pom would 
+        fix issue jcabi/jcabi-parent#19
```

Comment:
Let's add a single space here (`f` in line 104 should be aligned with `t`, not `@`).